charters/DataStore/charter.md

```diff
@@ -0,0 +1,75 @@
+
+# Data Store
+
+
+## Description
```

Comment:
*Minor:*
* Even Amazon _spells out_ PB as either _petabyte scale_ or _petabyte-scale_. 
* What is the difference between _strong public APIs_ and _simple API_? What is meant by _strong_ for example?

Perhaps *... on multiple clouds at petabyte-scale, ~~with strong public APIs~~. It defines public APIs for storage ...*

Reply:
Will do, thanks.